lib/external-parsers/base.ts

```diff
@@ -50,11 +51,17 @@ export class ExternalParserBase implements IExternalParser {
     private getObjdumpStarterScriptContent(filters: ParseFiltersAndOutputOptions) {
         const parserArgs = this.getParserArguments(filters, true);
 
+        let winePrefix = "";
```

Comment:
When the objdump is an exe it would not work. I'm not sure if this is the best way to find out if the objdump is an exe, please let me know if there is a better way.

Reply:
Winedebug things are probably best to leave to CE itself, but there probably isn't a way to set `WINEDEBUG` at the moment (although it's slightly insane wine requires this to run this, but that's a different can of worms probably)

Reply:
Should also probably check if this is running on linux as well.